Source/Core/VideoCommon/OpcodeDecoding.cpp

```diff
@@ -63,15 +63,18 @@ class RunCallback final : public Callback
       {
         VertexLoaderManager::g_main_vat_dirty = BitSet8::AllTrue(CP_NUM_VAT_REG);
         VertexLoaderManager::g_bases_dirty = true;
+        VertexLoaderManager::g_cp_xf_dirty = true;
       }
       else if (sub_command == CP_VAT_REG_A || sub_command == CP_VAT_REG_B ||
                sub_command == CP_VAT_REG_C)
       {
         VertexLoaderManager::g_main_vat_dirty[command & CP_VAT_MASK] = true;
+        VertexLoaderManager::g_cp_xf_dirty = true;
       }
       else if (sub_command == ARRAY_BASE)
       {
         VertexLoaderManager::g_bases_dirty = true;
+        VertexLoaderManager::g_cp_xf_dirty = true;
```

Comment:
`CheckCPConfiguration` doesn't care about the array bases. It does, however, care about `matrix_index_a` and `matrix_index_b`.